datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxHorizontalScalingService.java

```diff
@@ -99,24 +107,28 @@ public String triggerScalingFlow(SdxCluster sdxCluster, StackScaleV4Request scal
     }
 
     public void validateHorizontalScaleRequest(SdxCluster sdxCluster, DatalakeHorizontalScaleRequest request) {
-        if (!ENTERPRISE.equals(sdxCluster.getClusterShape())) {
-            throw new BadRequestException(String.format("scaling not supported on: %s. " +
-                    "Only the ENTERPRISE Datalake shape support the horizontal scaling functionality.", sdxCluster.getClusterShape().name()));
+        if (ENTERPRISE != sdxCluster.getClusterShape()) {
+            throw new BadRequestException(String.format("scaling not supported on: %s.", sdxCluster.getClusterShape().name()));
         }
         String hostGroup = request.getGroup();
         if (!DatalakeInstanceGroupScalingDetails.valueOf(hostGroup.toUpperCase(Locale.ROOT)).isScalable()) {
             throw new BadRequestException(String.format("%s host group cannot scale", hostGroup));
         }
         DetailedEnvironmentResponse environmentResponse = environmentClientService.getByName(sdxCluster.getEnvName());
         if (!EnvironmentStatus.upscalable().contains(environmentResponse.getEnvironmentStatus())) {
-            throw new BadRequestException(String.format("The DAtalake cannot be scaled because the environment %s is in the %s status.",
-                    sdxCluster.getEnvName(), environmentResponse.getEnvironmentStatus().name()));
+            throw new BadRequestException(String.format("The cluster cannot be scaled in the %s environment status.",
```

Comment:
You are using various naming, once it's cluster once its Datalake.
I would prefer the datalake as this comes from the DL context and it's easier to pinpoint the error message.